pennylane/qinfo/transforms.py

```diff
@@ -646,7 +646,10 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def quantum_fisher(qnode, *args, **kwargs):
+@partial(transform, is_informative=True)
+def quantum_fisher(
+    tape: qml.tape.QuantumTape, device, *args, **kwargs
```

Comment:
I think it's better to put device in `**kwargs` so that it doesn't render in the docs. That way it's more obvious that the user isn't supposed to provide a device manually.

Reply:
It seems like we don't care about that. Nevermind :)